index.js

```diff
@@ -30,25 +30,16 @@ module.exports = {
     if (type === 'parent') {
       this._assignOptions(this.app);
 
-      if (this._stripTestSelectors) {
-        let StripTestSelectorsTransform = require('./strip-test-selectors');
-
-        registry.add('htmlbars-ast-plugin', {
-          name: 'strip-test-selectors',
-          plugin: StripTestSelectorsTransform,
-          baseDir() { return __dirname; },
-          cacheKey() { return 'strip-test-selectors'; },
-        });
-      } else {
-        let TransformTestSelectorParamsToHashPairs = require('./transform-test-selector-params-to-hash-pairs');
-
-        registry.add('htmlbars-ast-plugin', {
-          name: 'transform-test-selector-params-to-hash-pairs',
-          plugin: TransformTestSelectorParamsToHashPairs,
-          baseDir() { return __dirname; },
-          cacheKey() { return 'transform-test-selector-params-to-hash-pairs'; },
-        });
-      }
+      let pluginFnName = this._stripTestSelectors ? '_buildStripPlugin' : '_buildHashParamPlugin';
+
+      let plugin = this[pluginFnName]();
+      plugin.parallelBabel = {
```

Comment:
@Turbo87 does this need to be `_parallelBabel`?

https://github.com/babel/broccoli-babel-transpiler/search?q=_parallelBabel&unscoped_q=_parallelBabel

Reply:
no, because it's for the HTMLBars transform, not Babel... wish they would have the same api...